src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a commandline utility designed to simplify the deployment and maintenance of
+submariner across your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+The subctl program works with [submariner-operator](https://github.com/submariner-io/submariner-operator).
+It helps automate the operator deployment process reducing the possibility of mistakes during
+the process.
+
+Subctl connects to your cluster(s) and performs the requested *command*. For the `join`
+command it will deploy the submariner-operator.
+
+### Commands
+
+#### deploy-broker
+
+`subctl deploy-broker [flags]`
+
+The **deploy-broker** command will configure the cluster pointed out by the `--kubeconfig`
+(or `KUBECONFIG` env var) and the `--kubecontext` flag as the broker. It will install
+the necessary CRDs and the `submariner-k8s-broker` namespace.
+
+It will generate the `broker-info.subm` file which can be used with the `join` command
+to connect clusters to the broker. This file contains the following details:
+
+ * Encryption PSK key
+ * Broker access details for subsequent `subctl` runs.
+ * Service discovery settings
+ * Globalnet settings
+
+##### deploy-broker flags
+
+    -h, --help                           help for deploy-broker
+
+    --kubeconfig string                  absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
+
+    --kubecontext string                 kubeconfig context to use
+
+    --service-discovery                  Enable Multi Cluster Service Discovery
+
+    --globalnet                          Enable support for Overlapping cluster/service CIDRs in connecting clusters (default disabled)
+
+    --globalnet-cidr-range string        Global CIDR supernet range for allocating GlobalCIDRs to each cluster
+                                         (default "169.254.0.0/16")
+
+    --ipsec-psk-from string              Import IPSEC PSK from existing submariner broker file, like broker-info.subm
+                                         (default "broker-info.subm)
+
+    --dataplane                          Install the Submariner dataplane on the broker
+                                         If this flag is enabled, the broker will be joined as if a join command 
+                                         was run right after deploy-broker. Use the join flags too if you use
+                                         --dataplane.
```

Comment:
This (somehow) renders as sort of a table with black background in a teletype font. However I don't think it presents nor reads well, eg the description text does not wrap on the column boundary. I think it would look better to use a regular table, eg:

| Flag              | Description
|:------------|:-----------|
| -h, --help    | help for deploy-broker|

Reply:
Yeah, may be we can try that format.

I believe it would work better if the code coloring javascript didn't trigger for this (does not make sense).

Reply:
I tried it ... and looks better on the other form (I know, its highly subjective).

![image](https://user-images.githubusercontent.com/1176434/81384461-fccd3580-9111-11ea-81f4-f9a560e0744e.png)

We need to find a way of fixing the coloring so it does not happen for non-code.

##### deploy-broker flags

| Flag                             | Description                                                                 
|:---------------------------------|:----------------------------------------------------------------------------|
|--kubeconfig <string>             | absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
|--kubecontext <string>            | kubeconfig context to use
|--service-discovery               | Enable Multi Cluster Service Discovery
|--globalnet                       | Enable support for Overlapping cluster/service CIDRs in connecting clusters (default disabled)
|--globalnet-cidr-range <string>   | Global CIDR supernet range for allocating GlobalCIDRs to each cluster (default "169.254.0.0/16")
|--ipsec-psk-from <string>         | Import IPSEC PSK from existing Submariner broker file, like broker-info.subm (default "broker-info.subm)
|--dataplane                       | Install the Submariner dataplane on the broker. If this flag is enabled, the broker will be joined as if a join command was run right after deploy-broker. Use the join flags too if you use --dataplane.

Reply:
This ^^ (aka table) looks good for me at least, instead of black background.

Reply:
LGTM too.